test/presets/netlify.test.ts

```diff
@@ -86,6 +118,24 @@ describe("nitro:preset:netlify", async () => {
           }
         `);
       });
+      it("writes server/server.mjs with static paths excluded", async () => {
+        const serverFunctionFile = await fsp.readFile(
+          resolve(ctx.outDir, "server/server.mjs"),
+          "utf8"
+        );
+        expect(serverFunctionFile).toEqual(
+          `
+export { default } from "./main.mjs";
+export const config = {
+  name: "server handler",
+  generator: "mock-framework@1.2.3",
+  path: "/*",
+  excludedPath: ["/.netlify/*","/icons/*","/nested/fonts/*","/build/*"],
```

Comment:
@pi0 where is `/build/*` coming from here? I couldn't quite trace it. I'd like to configure it explicitly in the test setup if possible.

Reply:
test fixture i suppose